### ISSUE TITLE
remove owner_id from planets response

### ISSUE THREAD

**Issue by a-tal** [ccpgames/esi-issues#712]
# Inconsistency

generally, excluding bulk endpoints, ESI does not return input parameters in the response objects

### Routes

/characters/{character_id}/planets/ (v1)

### Description

there is an `owner_id` property on each planet returned, always set to the character ID who made the request

### Resolution

remove owner_id in v2

**Comment by CarbonAlabel** [comment]
Might want to do the same for https://esi.tech.ccp.is/ui/?version=dev#/Corporation/get_corporations_corporation_id_structures

**Comment by Seavert** [comment]
Indeed, shouldn't `corporation_id` be removed from `/v3/corporations/{corporation_id}/structures/` before promotion?
